spec/fluent.ebnf

```diff
@@ -3,13 +3,24 @@
 Resource            ::= (blank_line | Entry | junk_line)*
 Entry               ::= Message
                       | Term
-                      | (ResourceComment | GroupComment | Comment)
-Message             ::= Comment? Identifier inline_space? "=" inline_space? ((Pattern Attribute*) | (Attribute+)) line_end
+                      | Section
+                      | ValueLessMessage
+                      | (ResourceComment | GroupComment | Comment | LegacyComment)
+Message             ::= (Comment | LegacyComment)? Identifier inline_space? "=" inline_space? ((Pattern Attribute*) | (Attribute+)) line_end
 Term                ::= Comment? TermIdentifier inline_space? "=" inline_space? Value Attribute* line_end
 Comment             ::= ("#" comment_line)+
 GroupComment        ::= ("##" comment_line)+
 ResourceComment     ::= ("###" comment_line)+
 
+/* "//"=style comments got removed in 0.5 */
```

Comment:
A stylistic nit of a non-native speaker: `were removed` reads better to me, even though I'm not really sure which one is better English.